src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMSource.java

```diff
@@ -172,7 +172,8 @@ public String getProjectKey() {
     }
 
     public String getProjectName() {
-        return getBitbucketSCMRepository().getProjectName();
+        BitbucketSCMRepository repository = getBitbucketSCMRepository();
+        return repository.isPrivate() ? repository.getProjectKey() : repository.getProjectName();
```

Comment:
I'm not sure why we called this `isPrivate`. Shouldn't it be `isPersonal`?

Reply:
Good catch, will refactor.